homeassistant/components/homewizard/coordinator.py

```diff
@@ -43,11 +45,21 @@ async def _async_update_data(self) -> DeviceResponseEntry:
                 data=await self.api.data(),
             )
 
-            if self.supports_state(data.device):
-                data.state = await self.api.state()
-
-            if self.supports_system(data.device):
-                data.system = await self.api.system()
+            try:
+                if self.supports_state(data.device):
+                    data.state = await self.api.state()
+
+                if self.supports_system(data.device):
+                    data.system = await self.api.system()
+
+            except UnsupportedError as ex:
+                # Old firmware, ignore
+                if not self._unsupported_error:
+                    self._unsupported_error = True
+                    _LOGGER.warning(
+                        "Feature not supported, make sure your device is up-to-date: %s",
```

Comment:
This is a bit unclear in wording; one could ask, "What feature?" or "What device?".

Should this be resolved by the user? Or, doesn't it matter at all? What would be the consequence?

Reply:
Good feedback, thanks!

I've updated the warning to make it more clear for the user. What do you think?

Example: `Energy Socket Kitchen is running an outdated firmware version (state is not supported). Contact HomeWizard support to update your device`

A repair would be an even better option, but there are _very_ little devices active that use the outdated version. We can add this if we get to many users with this issue, in my opinion.